### DISCUSSION TITLE
multi_cell string part(sub-string) make bold or add colors...

### DISCUSSION THREAD

**Discussion by akash-exactspace** [py-pdf/fpdf2#617] [Q&A] [answered]
If we want to make a part of a string(sub string) in multi_cell as bold italic or something also for highlighting purpose i may want to use some colors, Is there any way for doing this ? As documentation is not updated with any such conditions..........

**Comment by Lucas-C** [comment, marked as answer]
Hi @akash-exactspace

`FPDF.multi_cell()` has a `markdown` parameter that can be set to `True` to allow setting **\*\*bold\*\*** or _\_\_italic\_\__ text: https://pyfpdf.github.io/fpdf2/fpdf/fpdf.html#fpdf.fpdf.FPDF.multi_cell

Regarding colors, this is not currently possible with `multi_cell()`, but you can use `FPDF.write_html()`: https://pyfpdf.github.io/fpdf2/HTML.html

```python
pdf.write_html("<font color="#00ff00"><p>hello in green</p></font>")
```

**Reply by akash-exactspace** [reply]
Thank You Lucas.
